Ktisis/Interface/Components/BoneTree.cs

```diff
@@ -7,14 +7,31 @@
 using Ktisis.Structs;
 using Ktisis.Structs.Bones;
 using Ktisis.Structs.Actor;
+using Ktisis.Util;
+using Dalamud.Interface;
+using System;
 
 namespace Ktisis.Interface.Components {
 	public class BoneTree {
+		private enum HighlightReason {
+			None = 0,
+			Selected = 1,
+			Queried = 2,
+			ChildSelected = 4,
+			ChildQueried = 16
+		}
+
 		private static Vector2 _FrameMin;
 		private static Vector2 _FrameMax;
 
+		private static string SearchText = "";
+
 		public static unsafe void Draw(Actor* actor) {
```

Comment:
[IDE0036](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0036) was raised in accordance with [`.editorconfig`](https://github.com/ktisis-tools/Ktisis/blob/238952ca54ea5fc5cfbab339e66402ab89123fff/.editorconfig#L8).
```suggestion
		public unsafe static void Draw(Actor* actor) {
```